frontend/nuxt.config.js

```diff
@@ -23,14 +26,42 @@ export default {
 
     // Global page headers: https://go.nuxtjs.dev/config-head
     head: {
-        title: 'Auctions UI',
+        title: 'Unified Auctions UI',
         htmlAttrs: {
             lang: 'en',
         },
         meta: [
             { charset: 'utf-8' },
             { name: 'viewport', content: 'width=device-width, initial-scale=1' },
-            { hid: 'description', name: 'description', content: '' },
+            {
+                hid: 'description',
+                name: 'description',
+                content: SITE_DESCRIPTION,
+            },
+            { property: 'og:site_name', content: 'Unified Auctions UI' },
+            { hid: 'og:type', property: 'og:type', content: 'website' },
+            {
+                hid: 'og:url',
+                property: 'og:url',
+                content: process.env.PRODUCTION_DOMAIN,
```

Comment:
That's not necessarily true and also may contain the domain without `https://`, so it's not url, right?

Reply:
Your right! I removed this property as we also did not really need it.